src/main/java/com/github/black0nion/blackonionbot/commands/bot/SupportCommand.java

```diff
@@ -1,32 +1,33 @@
 package com.github.black0nion.blackonionbot.commands.bot;
 
+import com.github.black0nion.blackonionbot.commands.SlashCommand;
+import com.github.black0nion.blackonionbot.commands.SlashCommandEvent;
 import com.github.black0nion.blackonionbot.wrappers.jda.BlackGuild;
 import com.github.black0nion.blackonionbot.wrappers.jda.BlackMember;
 import com.github.black0nion.blackonionbot.wrappers.jda.BlackUser;
-import com.github.black0nion.blackonionbot.commands.TextCommand;
-import com.github.black0nion.blackonionbot.commands.CommandEvent;
-import net.dv8tion.jda.api.entities.Message;
 import net.dv8tion.jda.api.entities.TextChannel;
-import net.dv8tion.jda.api.events.message.MessageReceivedEvent;
+import net.dv8tion.jda.api.events.interaction.command.SlashCommandInteractionEvent;
+import net.dv8tion.jda.api.interactions.commands.build.Commands;
+import org.jetbrains.annotations.NotNull;
 
 import java.util.stream.Collectors;
 
 import static com.github.black0nion.blackonionbot.utils.config.Config.metadata;
 
-public class SupportCommand extends TextCommand {
+public class SupportCommand extends SlashCommand {
 
 	public SupportCommand() {
-		this.setCommand("support", "authors");
+		super(builder(Commands.slash("support", "Used to get support.")));
```

Comment:
If you want, you can use `super("support", "Used to get support.")` directly, but this is also completely fine